src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,36 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String modelCare;
+    private int carSpeed;
+    private String carColor;
+    private int weight;
+    private Engine engine;
 
+    public Car() {
+    }
+
+    public Car(String modelCare, int carSpeed, String carColor, int weight, Engine carEngine) {
+        this.modelCare = modelCare;
+        this.carSpeed = carSpeed;
+        this.carColor = carColor;
+        this.weight = weight;
+        this.engine = carEngine;
+    }
+
+    public Engine getEngine() {
+        return engine;
+    }
+
+    public void setEngine(Engine engine) {
+        this.engine = engine;
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
+        Car newCar = (Car) super.clone();
```

Comment:
please read the checklist before sending the task for review

Reply:
Sorry, I will be attentive.